tests/VolleyManagement.Domain.UnitTests/TeamTests.cs

```diff
@@ -205,15 +205,171 @@ public void Team_RosterContainsAlreadyTheSamePlayerId_ExceptionIsThrown()
             //Act
             Action act = () =>
             {
-                team.Roster = new List<PlayerId>() {
-            new PlayerId(2),
-            new PlayerId(2)
-                    };
+                team.AddPlayers(new List<PlayerId>() {
+                    new PlayerId(2),
+                    new PlayerId(2)
+                });
             };
 
             //Assert
             act.Should().Throw<ArgumentException>("Team Roster can`t has players with the same Id.");
         }
+
+        [Fact]
+        public void Team_RosterHasAlreadyThisPlayer_ExceptionIsThrown()
+        {
+            //Arrange
+            var team = CreateTeam();
+            team.AddPlayers(new List<PlayerId>() {
+                new PlayerId(3),
```

Comment:
Do not use magic numbers. Encapsulate those statements into methods with descriptive names